app/src/main/kotlin/com/simplemobiletools/smsmessenger/extensions/Context.kt

```diff
@@ -993,3 +995,12 @@ fun Context.clearAllMessagesIfNeeded() {
         config.wasDbCleared = true
     }
 }
+
+fun Context.subscriptionManagerCompat(): SubscriptionManager {
+    return if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.M) {
```

Comment:
lets use IsMarshmallowPlus or whatever is it called

Reply:
[done :+1: ](https://github.com/SimpleMobileTools/Simple-SMS-Messenger/pull/431/commits/f4575401f59db32e8739261bf5bed5a5be088580)